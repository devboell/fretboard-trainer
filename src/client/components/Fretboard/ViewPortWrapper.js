import styled from 'styled-components'

const ViewPortWrapper = styled.div`
  width: 100%;
  height: ${props => props.height}px;
`

export default ViewPortWrapper
