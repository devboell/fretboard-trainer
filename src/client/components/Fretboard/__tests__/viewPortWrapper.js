import React from 'react'
import 'jest-styled-components'

import ViewPortWrapper from '../ViewPortWrapper'

const props = {
  height: 40,
}

it('should return ViewPortWrapper with correct height', () => {
  const wrapper = shallow(<ViewPortWrapper {...props} />)
  expect(wrapper).toMatchSnapshot()
})
