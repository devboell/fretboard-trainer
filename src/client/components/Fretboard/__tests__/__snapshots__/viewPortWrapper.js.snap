// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return ViewPortWrapper with correct height 1`] = `
.c0 {
  width: 100%;
  height: 40px;
}

<div
  className="c0"
  height={40}
/>
`;
